jib-cli/src/test/java/com/google/cloud/tools/jib/cli/jar/JarModeProcessorTest.java

```diff
@@ -326,6 +326,125 @@ public void testExplodedMode_standard_computeEntrypoint_noMainClass() throws URI
     assertThat(ex)
         .hasMessageThat()
         .isEqualTo(
-            "`Main-Class:` attribute for an application main class not defined in the input Jar's manifest (`META-INF/MANIFEST.MF` in the Jar).");
+            "`Main-Class:` attribute for an application main class not defined in the input JAR's manifest (`META-INF/MANIFEST.MF` in the JAR).");
+  }
+
+  @Test
+  public void testCreatePackagedModeLayersForStandardJar_emptyJar()
+      throws IOException, URISyntaxException {
+    Path standardJar = Paths.get(Resources.getResource(STANDARD_JAR_EMPTY).toURI());
+    List<FileEntriesLayer> layers =
+        JarModeProcessor.createPackagedModeLayersForStandardJar(standardJar);
+
+    assertThat(layers.size()).isEqualTo(1);
+
+    FileEntriesLayer jarLayer = layers.get(0);
+    assertThat(jarLayer.getName()).isEqualTo("jar");
+    assertThat(
+            jarLayer
+                .getEntries()
+                .stream()
+                .map(FileEntry::getExtractionPath)
+                .collect(Collectors.toList())
+                .get(0))
```

Comment:
`jarLayer().getEntries().get(0).getExtractionPath()`?

And before this, also assert that the size of `jarLayer().getEntries()` is 1.